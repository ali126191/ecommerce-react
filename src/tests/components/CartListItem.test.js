import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import CartListItem from '../../components/CartListItem';

test('should render CartListItem correctly', () => {
    const wrapper = shallow(<CartListItem />);
    expect(toJSON(wrapper)).toMatchSnapshot()
})