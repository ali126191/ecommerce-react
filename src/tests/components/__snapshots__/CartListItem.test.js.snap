// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CartListItem correctly 1`] = `
<div>
  Item
</div>
`;
